website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -792,9 +792,11 @@ The refined ORCA inventory catalog build script is available for download <a hre
 ### Addressing open questions
 
 - What is the proper date to use when receiving a date range from Cumulus for a reconciliation report?
-  - After discussion with the Cumulus team, we have decided to use `cumulus_createdAt` timestamp.
+  - After discussion with the Cumulus team, we have decided to use `cumulus_create_time` 
+    which is passed by Cumulus in the CMA Message containing the timestamp that the granule began ingest of the granule into Cumulus.
```

Comment:
may want to add  that the value comes from the `createdAt` key in the CMA.

Reply:
added